### DISCUSSION TITLE
Calendar is it available in the future

### DISCUSSION THREAD

**Discussion by ab2d3e8** [twbs/bootstrap#33693] [Ideas]
I expect this Calendar is necessary because they are related to time, so is it possible to see in bootstrap?

**Comment by JustCallMeAD** [comment]
Checkout https://fullcalendar.io/ as it has a Bootstrap theme.

**Comment by lorvent** [comment]
I am really against the idea because people will start asking for many features in it and it makes bs5 heavy.

**Reply by hax18** [reply]
Agree
